ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx

```diff
@@ -46,33 +38,12 @@ const NotificationGroupNotification = ({ id, type }: Props): JSX.Element => {
         data-test="notification-message"
         dangerouslySetInnerHTML={{ __html: notification.message }}
       ></span>
-      {showMenu && (
-        <ContextualMenu
-          className="p-notification__menu-button"
-          dropdownClassName="p-notification__menu"
-          dropdownContent={
-            <>
-              <div className="u-flex--between">
-                <div className="u-sv1">Enable new release notifications</div>
-                <Switch
-                  defaultChecked={Boolean(releaseNotifications)}
-                  onChange={(evt: React.ChangeEvent<HTMLInputElement>) => {
-                    dispatch(
-                      configActions.update({
-                        release_notifications: evt.target.checked,
-                      })
-                    );
-                  }}
-                />
-              </div>
-              <Link to="/settings/configuration/general">See settings</Link>
-            </>
-          }
-          hasToggleIcon
-          position="right"
-          toggleAppearance="base"
-        />
-      )}
+      {showSettings ? (
+        <>
+          {" "}
+          <Link to="/settings/configuration/general">See settings</Link>
```

Comment:
Shouldn't the link text and to be props of this component?

Reply:
This component checks for new release notifications and this link is only shown if it is one. This component is really a wrapper around `<Notification>` with specific use in the top notification list.

Reply:
Hope that made sense :).